### PR TITLE
Update styling:

### PR DESCRIPTION
**Pull request by taylorjdawson**
- Fix rankings not rendering
  - Fix fonts to match figma design
  - Respect the display style of the asset (padded, contained, covered, etc.)
  - The collection bubble now links to the collection

### CODE REVIEW

**Comment by taylorjdawson** [comment]
image

**Comment by taylorjdawson** [comment]
@alexanderatallah one question I have is how best to handle `card_display_style` `padded`

**Comment by alexanderatallah** [comment]
@taylorjdawson properties, stats, and boosts still look like 3 different fonts (both 3 diff weights and 3 diff sizes!)

**Comment by alexanderatallah** [comment]
The asset name ("W10" above) could also be the same font as the price, which looks slightly heavier

**Comment by taylorjdawson** [comment]
> @taylorjdawson properties, stats, and boosts still look like 3 different fonts (both 3 diff weights and 3 diff sizes!)

@alexanderatallah 

How's this?
image

File: example/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <title>OpenSea Embeddable NFT example</title>
     <style media="screen">
         body {
-          font-family: Avenir,Helvetica,Arial,sans-serif;
+          /*font-family: Avenir,Helvetica,Arial,sans-serif;*/
```
Comment (alexanderatallah) [inline example/index.html]: let's remove this

File: src/nft-card.ts [hunk 2]
```diff
@@ -80,7 +80,8 @@ export class NftCard extends LitElement {
       }
       .card {
         background-color: transparent;
-        font-family: Avenir Next, Avenir, Helvetica Neue, sans-serif;
+        font-family: 'Roboto', sans-serif;
```
Comment (alexanderatallah) [inline src/nft-card.ts]: where else is Roboto being used? we're not `<link>`ing to the googlefonts link for it right? I don't think all operating systems ship with it

Comment (taylorjdawson) [reply src/nft-card.ts]: Yeah it is imported [here](http://github.com/ProjectOpenSea/embeddable_nfts/blob/0e3ef2516879ada8f02bbf916cbea9a9bbd252a4/src/nft-card.ts#L231-L231)

**Pull request merged**